### ISSUE TITLE
Question in 02_Tensor/03_multiple_... , feed_dict={x_data=x_val}

### ISSUE THREAD

**Issue by leigaoyi** [nfmcclure/tensorflow_cookbook#23]
First, I change the type of x_val with np.float32 into y_val. When I run:
 **mov_avg_layer= tf.nn.conv2d(y_val, my_filter, my_strides,
                           padding='SAME', name='Moving_Avg_Window')
print(sess.run(mov_avg_layer))**
I get the result. But when I try the feed_dict, my kernel runs out.

When I put the codes in my terminal python envirment, my operation system runs wrong.
I use the Gpu size of tensorflow, did this double? Or other things make my feed_dict goes out?
**mov_avg_layer= tf.nn.conv2d(x_data, my_filter, my_strides,
                           padding='SAME', name='Moving_Avg_Window')
print(sess.run(mov_avg_layer,feed_dict={x_data:y_val}))**
The above command make my kernel die without any information.

**Comment by nfmcclure** [comment]
Hi @leigaoyi ,  This is almost surely due to the TF v1+ upgrade.  I went through a few of the scripts tonight and made sure they run with version 1.0.1.  This is now running for me.  I recommend making sure you are running version 1.0.1, and retrying this script or notebook.  Let me know if you are still running into errors.
